Kristian-Repizo/pokeCollection/src/containers/GameInfoContainer.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import GameDescription from '../components/GameDescription';
+import NavBar from '../components/NavBar';
+import Footer from '../components/Footer';
+
+import axios from 'axios';
+
+class GameInfoContainer extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            gameDescription: "",
+            gameName: "",
+            gameId: "",
+            habitat: "",
+            abilities: [],
+            moves: [],
+            height: "",
+            weight: "",
+            types: [],
+        }
+    }
+
+    componentDidMount() {
+        //Peticion con axios
+        const { match } = this.props;
+        const gameId = match.params.gameIndex;
+        const gameName = match.params.gameName;
+        const gameDescriptionUrl = `${process.env.REACT_APP_GAME_API_BASE_URL}pokemon-species/${gameId}/`;
+        this.getStats();
+
+        axios.get(gameDescriptionUrl)
```

Comment:
@Kristian-Repizo data should not be obtained from a component, all API calls and data logic should be in a Context